src/Context/ThemeContext.tsx

```diff
@@ -12,8 +12,8 @@ interface ThemeContextProviderProps {
 export const ThemeContext = createContext<ITheme | null>(null);
 
 const ThemeContextProvider: React.FC<ThemeContextProviderProps>= (props)=>{
-    const [theme, setTheme] = useState('light');
-
+    const [theme, setTheme] = useState(()=> localStorage.getItem("theme") || "light");
```

Comment:
we haven't set the theme item in local storage, so this is not doing anything.
If you switch to dark mode, and then refresh the page. The value will return to light mode, not dark mode.